test/factories/ips.rb

```diff
@@ -1,5 +1,35 @@
 FactoryBot.define do
   factory :ip do
-    address { "MyString" }
+    address { Faker::Internet.ip_v4_address }
+    continent_code { 'AS' }
+    continent_geoname_id { 6255147 }
+    continent_name { 'Asia' }
+    country_confidence { 99 }
+    country_geoname_id { 1814991 }
+    country_name { 'China' }
+    registered_country_iso_code { 'CN' }
+    registered_country_geoname_id { 1814991 }
+    registered_country_name { 'China' }
+    traits_anonymous { true }
+    traits_hosting_provider { true }
+    traits_user_type { 'hosting' }
+    traits_autonomous_system_number { 00000 }
+    traits_isp { 'Organization' }
+    traits_organization { 'Organization' }
+    location_time_zone { 'Asia/Shanghai' }
+
+    factory :ip_berlin do
```

Comment:
I think you shoul use trait here and below https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits